SITCOMTN-086.tex

```diff
@@ -619,23 +813,93 @@ \subsubsection{Reference Flux Flats}
 To handle lateral electric field effects we fit templates based on our physical
 knowledge of the detectors.  This includes radial templates for tree rings with
 a fixed center, and square templates around the edge of the detectors.  The
-exact methodology is TBD.
-
-To handle large scale variations for LSST Y1 we use a ``star flat'' generated
-from a sequence of multi-band dithered observations of a moderately dense star
-field.  These observations are cross-calibrated using FGCM to generate
-per-detector polynomial approximations of the throughput relative to the dome
-flat at scales of hundreds of pixels and greater.  By making use of stars of
-various colors, the FGCM solution can approximate the star flat terms for a
-flat reference spectrum.
-
-In the future, we plan to use what we call the ``dynamic'' CBP, where the
-synthetic stars from the CBP are dithered around the full focal plane.  The
-advantage of this is that the response can be more carefully controlled, and it
-can be performed in narrow wavelength steps to build up the dynamic CBP star
-flat with any arbitrary spectrum.  The software to perform this analysis will
-not be ready by the completion of construction and will be developed in the
-early years of the LSST.
+exact methodology is TBD, and may not be in place by the end of commissioning.
+
+To handle the larger scale illumination correction for LSST Y1 we use a ``star
+flat'' generated from a sequence of multi-band dithered observations of a
+moderately dense star field.  These observations are cross-calibrated using
+FGCM to generate per-detector polynomial approximations of the throughput
+relative to the dome flat at scales of tens of pixels and greater.  By
+making use of stars of various colors, the FGCM solution can approximate the
+star flat terms for a flat reference spectrum.
+
+The illumination correction is then generated per-band from the star flat and the
+pixel area variations determined from the Jacobian of the world coordinate
+system (WCS) from the camera model.  This becomes:
+%
+\begin{equation}
+  S^{\mathrm{illum}}(x, y) = \frac{1.0}{S^{\mathrm{star}}(x, y)\,\Omega(x, y)},
+\end{equation}
+%
+where $S^{\mathrm{illum}}(x, y)$ is the illumination correction,
+$S^{\mathrm{star}}(x, y)$ is the star flat, and $\Omega(x, y)$ is the pixel
+area field. The reference flux flat then becomes:
+%
+\begin{equation}
+  S^{\mathrm{refflux}}(x, y) = S^{\mathrm{dome}}(x, y)\,S^{\mathrm{illum}}(x,
+  y),
+\end{equation}
+%
+where $S^{\mathrm{dome}}(x, y)$ is the dome flat.
+
+While the reference flux flat is the flat to use to obtain a
+photometrically-flattened image, the dome flat is
+more appropriate to use to obtain a background-flattened image because the sky
+follows similar light paths, including scattering, as the Lambertian flat-field
+screen illuminator.   Thus, we approximate the background flat
```

Comment:
I got lost here, because I think this is the first reference to a background flat.  Can we explicitly say "we define a background flat S^bkg that is appropriate for background subtraction, and approximate that with the dome flat"?  If that's not correct, then I fear I'm more lost than I expected.